test/scenario_a_1.py

```diff
@@ -50,7 +49,7 @@ def __init__(self):
                                             self.plant.m,
                                             self.plant.g)
 
-        self.detection_time = [[] for _ in range(len(self.actuator_faults))]
+        self.detection_time = [[self.actuator_faults[i].time + self.fdi.delay] for i in range(len(self.actuator_faults))]
```

Comment:
```suggestion
        self.detection_time = [fault.time + self.fdi.delay for fault in self.actuator_faults]
```

이거 `data.h5`에 info 라던지로 저장이 되고 있나요?

Reply:
추후에 값을 사용할 수 있도록 info로 저장하는 것으로 수정하였습니다.